backend/README.md

```diff
@@ -4,115 +4,180 @@ Node.js app which subscribes to the blocks on the source chain and sends block d
+      "paraId": 2084,
+      "accountSeed": "//Alice//2084",
+      "feedId": 7
+    },
+    {
+      "httpUrl": "https://spiritnet.api.onfinality.io/public",
+      "paraId": 2086,
+      "accountSeed": "//Alice//2086",
+      "feedId": 8
+    },
+    {
+      "httpUrl": "https://basilisk.api.onfinality.io/public",
+      "paraId": 2090,
+      "accountSeed": "//Alice//2090",
+      "feedId": 9
+    },
+    {
+      "httpUrl": "https://altair.api.onfinality.io/public",
+      "paraId": 2088,
+      "accountSeed": "//Alice//2088",
+      "feedId": 10
+    },
+    {
+      "httpUrl": "https://parallel-heiko.api.onfinality.io/public",
+      "paraId": 2085,
+      "accountSeed": "//Alice//2085",
+      "feedId": 11
+    },
+    {
+      "httpUrl": "https://kintsugi.api.onfinality.io/public",
+      "paraId": 2092,
+      "accountSeed": "//Alice//2092",
+      "feedId": 12
+    }
+  ]
 }
 ```
 
 Where:
-* `targetChainUrl` - WebSocket JSON-RPC endpoint URL of the target (Subspace) chain where transactions with blocks will be sent
-* `downloadedArchivePath` - optional path to downloaded archive of blocks for a particular chain as RocksDB database (can be created with `tools/download-substrate-blocks` script)
-* `httpUrl` - HTTP JSON-RPC endpoint URL of a Substrate-based chain
-* `wsUrl` - WebSocket JSON-RPC endpoint URL of the main Substrate-based chain (in most cases relay chain like Kusama or Polkadot, but can be used with any other chain too)
-* `paraId` - ID of a parachain or parathread under above relay chain
-* `accountSeed` - seed for the account that will be used on target chain for submitting transactions with blocks for particular chain (all such accounts can be funded with `tools/fund-accounts` script)
-* `feedId` - ID of the feed already created on Subspace chain into which archived blocks will go (`tools/create-feeds` script can be used to create feeds for accounts in the config file)
+
+- `targetChainUrl` - WebSocket JSON-RPC endpoint URL of the target (Subspace) chain where transactions with blocks will be sent
+- `downloadedArchivePath` - optional path to downloaded archive of blocks for a particular chain as RocksDB database (can be created with `tools/download-substrate-blocks` script)
+- `httpUrl` - HTTP JSON-RPC endpoint URL of a Substrate-based chain
+- `wsUrl` - WebSocket JSON-RPC endpoint URL of the main Substrate-based chain (in most cases relay chain like Kusama or Polkadot, but can be used with any other chain too)
+- `paraId` - ID of a parachain or parathread under above relay chain
+- `accountSeed` - seed for the account that will be used on target chain for submitting transactions with blocks for particular chain (all such accounts can be funded with `tools/fund-accounts` script)
+- `feedId` - ID of the feed already created on Subspace chain into which archived blocks will go (`tools/create-feeds` script can be used to create feeds for accounts in the config file)
```

Comment:
Not sure why was this necessary, `*` is more common from what I have seen and is rendered exactly the same as `-`